experiments/raja-perf/experiment.py

```diff
@@ -56,6 +82,23 @@ def compute_applications_section(self):
         else:
             self.add_experiment_variable("n_ranks", n_resources, True)
 
+        if isinstance(n_resources, list):
+            size = [int(total_size / res) for res in n_resources]
+        else:
+            size = total_size
+        self.add_experiment_variable("size", size, True)
+
+        self.add_experiment_variable("repfact", self.spec.variants["repfact"][0], True)
+
+        # rajaperf_variants = self.spec.variants["variants"][0].replace("-", " ")
```

Comment:
Should the commented out code be removed?

Reply:
It has been removed.